lib/foreman_scc_manager/engine.rb

```diff
@@ -22,8 +22,8 @@ class Engine < ::Rails::Engine
 
     initializer 'foreman_scc_manager.register_plugin', :before => :finisher_hook do |_app|
       Foreman::Plugin.register :foreman_scc_manager do
-        requires_foreman '>= 1.22'
-        requires_foreman_plugin 'katello', '>= 3.12.0'
+        requires_foreman '>= 2.1'
+        requires_foreman_plugin 'katello', '>= 3.16.0'
```

Comment:
Is this okay or shall we also set the versions to 2.3/3.18 ?

Reply:
Only if it is really incompatible and does not work with <2.3/3.18